web-platform/website/src/components/pages/account-settings/AccountSettingsPageContent.tsx

```diff
@@ -48,6 +71,9 @@ export function AccountSettingsContentFactory() {
     case AccountSettingsState.EditEmail:
       return <EditAccountEmailView />;
 
+    case AccountSettingsState.EditPassword:
+      return <EditAccountPasswordView showToast={showToast} />;
```

Comment:
Does not seem right we placing a specific prop in a generic factory method. Maybe display and timer should be props within EditAccountPasswordView.

Reply:
I'll move the toast to the change password view for success; makes more sense